website_event_share/views/website_event.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+    <template id="event_share" name="Share Button" inherit_id="website_event.event_details" active="False" customize_show="True">
+        <xpath expr="//h4[@t-field='event.address_id']" position="after">
+            <div class="oe_share s_share text-center">
+                  <h3>Share 
```

Comment:
Please put the "Share" word inside a `<span>` and put the whole `<div>` inside a independent template.

This way I can overwrite this template to place the share buttons in another point of the event page while still being able to `t-call` the share snippet itself.

Thanks!